CorinneTest/Person.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+namespace CorinneTest
+{
+    public class Person 
```

Comment:
You created an interface `IPerson` but didn't implement that interface here on the concrete class.

Reply:
Yeah I added those interfaces when I was searching how to build a factory but they won't be necessary for this project.